src/main/java/nl/tudelft/model/LevelFactory.java

```diff
@@ -89,11 +90,42 @@ public Level getLevel(int id) throws IllegalArgumentException {
                 return getLevel3();
             case 4:
                 return getLevel4();
+            case 5: 
+                return getBossLevel();
             default:
                 throw new IllegalArgumentException();
         }
     }
 
+    private Level getBossLevel() {
+        final int id = 5;
+        // Create Bubbles for level
+        LinkedList<AbstractBubble> bubbles = new LinkedList<>();
+
+        AbstractBubble bubble = new BacchelliBubble(resources, 200,
+                resources.getVwallImage().getWidth() );
+
+        bubbles.add(bubble);
+
+        LinkedList<AbstractWall> walls = wallInit();
+        {
```

Comment:
What's the point of these brackets?
Can this commented code be deleted?

< review